reg_tests/test_files/vofSlosh/vofSlosh.norm.gold

```diff
@@ -1,200 +1,200 @@
-3.121984735997264 1 0.001
```

Comment:
I was really struck by how much this initial residual changed between here and the new stabilized result.